examples/semi_supervised/plot_label_propagation_structure.py

```diff
@@ -24,7 +24,7 @@
 n_samples = 200
 X, y = make_circles(n_samples=n_samples, shuffle=False)
 outer, inner = 0, 1
-labels = -np.ones(n_samples)
+labels = np.full(n_samples, -1)
```

Comment:
This should be `-1.` Otherwise the dtype would not be equivalent,
```py
>>> np.ones(2)*-1
array([-1., -1.])
>>> np.full(2, -1)
array([-1, -1])
```
one is float, the other would be int. I suspect that is the reason for failing tests (in other changes)